site-builder/src/display.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{fmt::Display, io::stdout};
+
+use crossterm::{
+    cursor::{RestorePosition, SavePosition},
+    style::{Print, PrintStyledContent, Stylize},
+    terminal::{Clear, ClearType},
+};
+
+pub fn header<S: Display>(message: S) {
+    if cfg!(not(test)) {
+        crossterm::execute!(
+            stdout(),
+            PrintStyledContent(format!("\n{message}\n").green().bold()),
+        )
+        .unwrap();
+    }
+}
+
+pub fn error<S: Display>(message: S) {
+    if cfg!(not(test)) {
+        crossterm::execute!(
+            stdout(),
```

Comment:
Suggestion: Use `stderr` here.

Reply:
I don't see a change here, did you forget to push something?

Reply:
I haven't yet pushed  :)

Reply:
Ah, sorry, I was a bit hasty... 😅

Reply:
![](https://i.redd.it/d976p88wwkb31.gif)